www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -5,7 +5,7 @@
   <items>
     <general>
       <enabled type="BooleanField">
-        <default>0</default>
+        <default>1</default>
```

Comment:
Err, what?

Reply:
why not? It there to configure if it should serve configured services.

Reply:
I don’t understand. Is this global context or not? There’s no rationale in the description to make deviation from our defaults reasonable.

Reply:
Yes, it is global but it only affects configured things. For example, if a plugin adds a custom vhost, this one is still served.

Reply:
Hmm, sorry, clicked resolved by accident.

This suggest the template does things in a way that makes this more convenient, but people will only take advantage of this on new plugin installs and I don’t like the idea of opt-out even though nginx must be installed willingly.

I’d rather see some other solution to the problem that is quicker to deploy and more in line with what we have elsewhere. Or dropped if that is a minor issue that is just as well documented to be the way it works.

Cc @adschellevis

Reply:
@fichtner marked it as unresolved.

Currently the flag is silently ignored - 1.8 will enforce it so for new installation it will keep the behaviour. With this patch it allows the user to disable all his web pages and streams (the services itself is always enabled like haproxy because nginx is meant to become infrastructure for other plugins which need one and it will not bind anything if nothing is configured).

Reply:
@fichtner @fabianfrz I don't like the idea of opt-out either,  if this is a migration issue, it's probably better to solve it there.  Before attaching a lot of new plugins upon this one, it might be a good idea to discuss plans first. I'm a bit worried about a whole new dependancy chain around some plugins.

Reply:
Should I revert this line now?